app/controllers/contact/govuk/problem_reports_controller.rb

```diff
@@ -51,6 +51,8 @@ def create
 private
 
   def extract_return_path(url)
+    # rails does not pass invalid urls through
```

Comment:
Need the comment?

Reply:
There are a bunch of other comments in this file too. Unsure how we decide what is needed and what isn't?